envisage/tests/test_extension_point_changed.py

```diff
@@ -361,3 +361,14 @@ def test_remove_plugin(self):
         self.assertEqual([], listener.new.added)
         self.assertEqual([1, 2, 3], listener.new.removed)
         self.assertEqual(0, listener.new.index)
+
+    def test_extension_point_change_event_str_representation(self):
+        """ test string representation of the ExtensionPointChangedEvent class
+        """
+        from envisage.api import ExtensionPointChangedEvent
```

Comment:
Nit: Since this is test code, it should be okay to move this to the top? (Otherwise I'd think there is something dodgy that is causing an import at the top to be problematic, in which case this deserves a comment to explain)